server/routes/property.routes.js

```diff
@@ -195,4 +195,27 @@ router.post(
   PropertyController.search,
 );
 
+/**
+ * @swagger
+ * /property/available:
```

Comment:
Let's use available options here so that we don't confuse it with available properties.
```suggestion
 * /property/available-options:
```